charts/csm-authorization/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: csm-authorization
-version: 1.3.0
-appVersion: 1.3.0
+version: 1.3.1
+appVersion: 1.3.1
```

Comment:
This can be 1.4.0 to target next release.

Reply:
Updated.